src/contracts/proxies/Vault721.sol

```diff
@@ -154,6 +154,8 @@ contract Vault721 is ERC721EnumerableUpgradeable {
     address _to,
     uint256 _tokenId
   ) public override(ERC721Upgradeable, IERC721Upgradeable) {
+    if (_proxyRegistry[_to] != address(0)) revert NotWallet();
```

Comment:
Does the revert in `_afterTokenTransfer` not cause the transfer to fail?

Reply:
removed